docs/experiments-msmarco-v2-unicoil-tilde-expansion.md

```diff
@@ -26,31 +26,35 @@ First, we need to download and extract the MS MARCO V2 passage dataset with uniC
 
 ```bash
 # Alternate mirrors of the same data, pick one:
-wget https://rgw.cs.uwaterloo.ca/JIMMYLIN-bucket0/data/msmarco-v2-passage-unicoil-tilde-expansion-b8.tar -P collections/
-wget https://vault.cs.uwaterloo.ca/s/tb3m3J45HFJNAbq/download -O collections/msmarco-v2-passage-unicoil-tilde-expansion-b8.tar
+wget https://rgw.cs.uwaterloo.ca/JIMMYLIN-bucket0/data/msmarco-passage-v2-unicoil-tilde-expansion-b8.tar -P collections/
+wget https://vault.cs.uwaterloo.ca/s/tb3m3J45HFJNAbq/download -O collections/msmarco-passage-v2-unicoil-tilde-expansion-b8.tar
 
-tar -xvf collections/msmarco-v2-passage-unicoil-tilde-expansion-b8.tar -C collections/
+tar -xvf collections/msmarco-passage-v2-unicoil-tilde-expansion-b8.tar -C collections/
```

Comment:
just to confirm, we are doing `msmarco-passage-v2` here for this file only, right?

Reply:
Yes, this file was created before we standardized our convention. The underlying `tar` file decompresses to `msmarco-passage-v2`. But at least uniCOIL noexp and uniCOIL + TILDE are consistent.

I think It's too much work to go back and change everything... that would requiring renaming the underlying data files, re-compressing, re-uploading, etc.

Reply:
sg